timelock-agent/src/main/java/com/palantir/timelock/config/TimeLockRuntimeConfiguration.java

```diff
@@ -19,10 +19,7 @@
 @Value.Immutable
 public abstract class TimeLockRuntimeConfiguration {
 
-    @Value.Default
-    public PaxosRuntimeConfiguration paxos() {
-        return ImmutablePaxosRuntimeConfiguration.builder().build();
-    }
+    public abstract PaxosRuntimeConfiguration paxos();
```

Comment:
I don't think this should change. The contents of the runtime config aren't required and can fall back on the default.